include/utils/types.hpp

```diff
@@ -132,20 +132,29 @@ struct TelemetryConfig {
 struct Requirement {
     Requirement() = default;
     Requirement(const std::string& requirement_id_, size_t index_);
```

Comment:
These could be dropped now

Reply:
Ah I thought I removed them already, must have overlooked that